cloud/src/autogluon/cloud/scripts/image_serve.py

```diff
@@ -30,9 +31,11 @@ def transform_fn(model, request_body, input_content_type, output_content_type="a
         buf = BytesIO(request_body)
         data = np.load(buf, allow_pickle=True)
         image_paths = []
-        for i, bytes in enumerate(data):
-            im = Image.open(BytesIO(base64.b85decode(bytes)))
-            im_name = f'{i}.png'
+        for bytes in data:
+            im_bytes = base64.b85decode(bytes)
+            im_hash = hashlib.md5(im_bytes).hexdigest()
```

Comment:
I suggest using sha1 instead - this will reduce probability of collisions.
also please mark the row with `nosec` so it does not trigger security scanners
```python
 # nosec B303 - not a cryptographic use
```

Reply:
Can we also move this block into a function - it's reused below multiple times -- see dittos.